web/src/lib/components/asset-viewer/detail-panel.svelte

```diff
@@ -75,6 +78,10 @@
     if (newAsset.id && !isSharedLink()) {
       const data = await getAssetInfo({ id: asset.id });
       people = data?.people || [];
+      await getFaces({ id: asset.id }).then((data) => {
```

Comment:
`data` already comes with the faces for the asset. No need to query for them again.

Reply:
I don't think so. `data` contains the people, and their assigned faces, but not the unassigned faces.

Reply:
Is that so? We should change that in the server then.

Reply:
I added faces to the getAssetInfo endpoint, but didn't update the client sdks yet.

Reply:
Client sdks are not up-to-date.